app/controllers/cart_controller.rb

```diff
@@ -22,12 +22,30 @@ def remove_item
     redirect_to '/cart'
   end
 
+  def edit_quantity
+    adjust_quantity
```

Comment:
Put in helper method for the functionality which is now down in the private section under adjust_quantity